src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,75 @@
  */
 
 public class Plane {
+    private int modelNumber;
```

Comment:
try creating object of Plane (for this purpose create main class and method) and write your code in comment under this

Reply:
@Sofasmile  /*
        First of all we can use Plane constructor
        with all parameters 'cause without it I couldn't pass all testes(We've discuss it with Pavel
        on Q&a session. And also I'm using Plane builder,where I've created PlaneBuilder and used
        his methods(setters) to set some fields only(I don't have to set all fields like in case
        with Plane constructor) and then I'm using build() method to create the Plane object.
         */
        Plane plane = new Plane(23,45,26,34,"red");
        Plane.PlaneBuilder redPlaneBuilder =new Plane.PlaneBuilder();
        redPlaneBuilder.setColor("red").setModelNumber(24);
        Plane redPlaneObject = redPlaneBuilder.build();

Reply:
you should have ability to create object like this:
Plane plane = new Plane.PlaneBuilder().setWidth(12).build();

Reply:
@Sofasmile got it)